src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,3 @@
+.header-logo {
```

Comment:
Is it a separate bem block? Probably will be better to make it as header element and move it to `header.scss` file?